.github/workflows/helm-release.yml

```diff
@@ -1,7 +1,6 @@
 name: Release Helm chart
 on:
-  push:
-    branches: [main]
+  workflow_dispatch:
```

Comment:
I guess the `:` is not required, since there's no input for the workflow dispatch, buut I also think is harmless to have it like this